pom.xml

```diff
@@ -31,6 +31,24 @@
       </license>
     </licenses>
 
+    <repositories>
+      <repository>
+        <id>repo.jenkins-ci.org</id>
+        <url>http://repo.jenkins-ci.org/public/</url>
+      </repository>
+      <repository>
+          <id>jitpack.io</id>
+          <url>https://jitpack.io</url>
+      </repository>
+    </repositories>
+
+    <pluginRepositories>
+      <pluginRepository>
+        <id>repo.jenkins-ci.org</id>
+        <url>http://repo.jenkins-ci.org/public/</url>
+      </pluginRepository>
+    </pluginRepositories>
+
```

Comment:
Hmm, I guess I missed something but without these lines `mvn package` fails to build. Only with these lines it the project will build.

See https://gist.github.com/ssbarnea/5e7afb65cf4db3885c9814d2ed657cca

I think that you did not encounter these errors because your did pre-customize your mvn config.